.github/workflows/ci.yml

```
@@ -127,9 +127,11 @@ jobs:
           PYTHON: ''
           TRIXI_TEST: ${{ matrix.trixi_test }}
       - uses: julia-actions/julia-processcoverage@v1
+        if: always()
```

Since there is the warning 

```
Avoid using `always` for any task that could suffer from a critical failure, for example: getting sources, otherwise the workflow may hang until it times out. If you want to run a job or step regardless of its success or failure, use the recommended alternative: `if: ${{ !cancelled() }}`
```

at https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#always, would it be better to use

```suggestion
        if: ${{ !cancelled() }}
```

here and elsewhere?

---

Done!